cmudb/extensions/tscout/main.c

```diff
@@ -0,0 +1,224 @@
+#include <time.h>
+
+// clang-format off
+// Extension magic.
+#include "postgres.h"
+#include "fmgr.h"
+// clang-format on
+
+#include "commands/createas.h"
+#include "commands/explain.h"
+#include "optimizer/planner.h"
+#include "parser/parsetree.h"
+#include "utils/builtins.h"
+
+#include "../../../src/include/tscout/marker.h"
```

Comment:
You could do `tscout/marker.h` instead?

```suggestion
#include "tscout/marker.h"
```

Reply:
Oops. Yes! Thanks for this!